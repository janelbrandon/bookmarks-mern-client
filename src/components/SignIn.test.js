import React from 'react'
import SignIn from './SignIn'
import renderer from 'react-test-renderer'

describe('SignIn', () => {
  it('should render as expected', () => {
    const tree = renderer.create(
      <SignIn handleSignIn={ () => {} } loginError='There was an error'/>
    )
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
