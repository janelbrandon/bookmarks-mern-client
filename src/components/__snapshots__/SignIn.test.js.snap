// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn should render as expected 1`] = `
<form
  onSubmit={[Function]}
>
  <p>
    There was a login error
  </p>
  <label>
    Email: 
    <input
      name="email"
      type="email"
    />
  </label>
  <br />
  <label>
    Password: 
    <input
      name="password"
      type="password"
    />
  </label>
  <br />
  <button
    type="submit"
  >
    Login
  </button>
</form>
`;
